Cartfile.resolved

```diff
@@ -1,11 +1,12 @@
-github "Alamofire/Alamofire" "1.2.1"
+github "Alamofire/Alamofire" "1.2.2"
 github "norio-nomura/Base32" "v0.3.0"
 github "bignerdranch/Deferred" "4a5fa72bc460d08e261da810a4d040a407b38db0"
 github "swisspol/GCDWebServer" "3.2.3"
-github "kif-framework/KIF" "v3.2.1"
+github "kif-framework/KIF" "v3.2.2"
 github "rs/SDWebImage" "3.7.2"
 github "st3fan/SQLite.swift" "91839c620f3912eec83cb098c02213438cee480c"
 github "CEWendel/SWTableViewCell" "0.3.7"
 github "st3fan/SWXMLHash" "052bd278e88d024d83dcbd570bfce3b35de87573"
 github "Masonry/Snap" "0.11.0"
-github "DaveWoodCom/XCGLogger" "29b77cc64147aa09fe233ad2c90007fb4cd1fbf5"
+github "DaveWoodCom/XCGLogger" "74390d672ed2256360ea48f289286718afc9efe6"
+github "sqlcipher/sqlcipher" "v3.3.0"
```

Comment:
I'm a bit nervous to use something that's dependent on another framework for SwiftData. Having SQLite.swift in the tree makes things... confusing here.